sys/net/gnrc/transport_layer/tcp/gnrc_tcp_fsm.c

```diff
@@ -148,11 +148,6 @@ static int _transition_to(gnrc_tcp_tcb_t *tcb, fsm_state_t state)
 #endif
             tcb->peer_port = PORT_UNSPEC;
 
-            /* Allocate receive buffer */
-            if (_rcvbuf_get_buffer(tcb) == -ENOMEM) {
```

Comment:
Maybe now that only `FSM_STATE_SYN_SENT` may fail, add this fact to the documentation of `_transition_to()`:

```
@return   -EADDRINUSE, if @p state == `FSM_STATE_SYN_SENT` and local addres is already in use
```